src/main/java/socialite/logic/commands/ShareCommand.java

```diff
@@ -33,7 +33,7 @@
     public static final String MESSAGE_SHARE_PERSON_TEMPLATE = "Here are the social media handles for %1$s!\n"
             + "%2$s\n"
             + "Hey, have you tried SociaLite? I have enjoyed using it, you should check it out too!\n"
-            + "https://ay2122s1-cs2103t-f11-4.github.io/tp/";
+            + "Simply go to https://tinyurl.com/CS2103SociaLite!";
```

Comment:
Might be better to leave out the exclamation mark at the end as the user might interpret the special character as part of the URL